src/main/resources/application.properties

```diff
@@ -8,4 +8,8 @@ spring.jpa.generate-ddl=true
 
 # JWT secret
 jwtSecret=secret
-jwtExpiration=36000000
\ No newline at end of file
+jwtExpiration=36000000
+
+# Python
+pythonInterpreter=C:/Users/NhatAnh/AppData/Local/Programs/Python/Python310/python.exe
+pythonProgramLocation=C:/Users/NhatAnh/Desktop/Projects/ProjectWeb2/src/controllers/recommender.py
```

Comment:
what does this script do exactly? can it be implemented in java?

Reply:
this python script predicts the rating that this current user will give to all products, and based on that, this script will output 5 products have the highest rating. 
it's inefficient to implement it in java as it requires a lot of matrix calculation, which is most suitable for python.